src/Adapter/CombinationDataProvider.php

```diff
@@ -29,6 +29,7 @@
 use Combination;
 use PrestaShop\Decimal\Number;
 use PrestaShop\PrestaShop\Adapter\Product\ProductDataProvider;
+use PrestaShop\PrestaShop\Core\Localization\Locale;
```

Comment:
With the new improvement you could simplify this class and inject the Locale directly Since it's an internal class I think we can change the constructor safely

Reply:
Fixed